graphics/vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,62 @@
+"""
+render 3d points for 2d surfaces.
+"""
+
+import math
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+
+def convert_to_2d(x: float, y: float, z: float, scale: float,
+                  distance: float) -> (float, float):
```

Comment:
```suggestion
                  distance: float) -> tuple[float, float]:
```
The first line of code should be `from __future__ import annotations__`

Should scale and distance be ints or floats?

Reply:
floats can be accepted, their is no problem when i tested